src/shared/components/TimeSeries.tsx

```diff
@@ -267,6 +269,7 @@ class TimeSeries extends Component<Props, State> {
 
       let statuses = [] as StatusRow[][]
       if (check) {
+        // DLW FIXME TODO: decide how to handle
```

Comment:
The below used of `buildUsedVarsOption` is with multiple queries at once.
Some of which may define the variable internally. Some of which may not.
So how do we handle when/if to include the windowPeriod variable?